scripts/openstack-quickstart-demosetup

```diff
@@ -264,7 +264,8 @@ EODASHDB
 SESSION_COOKIE_SECURE = CSRF_COOKIE_SECURE = USE_SSL
 EOSEC
     # sync dashboard DB "after" the database is created
-    run_as wwwrun "cd /srv/www/openstack-dashboard; umask 0027; python -m 'manage' syncdb --noinput"
+    run_as wwwrun "cd /srv/www/openstack-dashboard; umask 0027; python -m 'manage' makemigrations"
```

Comment:
Django will need to session, auth and other modules, and I think that horizon have also models too. You say that the migrations are not distributed, in that case makemigration is indeed needed.